.github/workflows/charmcraft-pack.yaml

```diff
@@ -42,6 +35,19 @@ jobs:
           CLONE_URL: ${{ github.event.pull_request.head.repo.clone_url }}
           CLONE_SHA: ${{ github.event.pull_request.head.sha }}
 
+      - name: Install yq
+        run: sudo snap install yq
+
+      - name: Add 'git' as a build package 
+        run: |
+          cd examples/httpbin-demo
+          if yq --exit-status 'has("parts")' charmcraft.yaml; then
+            echo "'parts' already exists in charmcraft.yaml"
+            exit 1
+          fi
+          yq --inplace '.parts.charm.build-packages = ["git"]' charmcraft.yaml
```

Comment:
nit: probably not great to whack old build packages in case one is added later, but then we'd notice very quickly that this job fails.